workshop/content/basics/templates/linting-and-testing/_index.md

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
I have updated some wording and did some markdown cleanup. Please reflect the changes done in `code/` here.